### PR TITLE
Pretty print

### LINKED ISSUE #93: Create an SCD block the tabs in OpenSCD 2.8.1 until save and re-open
Creating a SCD from scratch in OpenSCD 2.8.1 block all the tabs and buttons for create the substation.
 
When the project is saved and re-open then the tabs and buttons are still available

### PR DESCRIPTION
**Pull request by c-dinkel**
Closes #93 .

### CODE REVIEW

File: src/open-scd.ts [hunk 1]
```diff
@@ -130,11 +130,27 @@ export class OpenSCD extends Setting(
     this.saveUI.close();
   }
 
+  private formatXml(xml: string, tab?: string) {
+    let formatted = '',
+      indent = '';
+
+    if (!tab) tab = '\t';
+    xml.split(/>\s*</).forEach(function (node) {
```
Comment (c-dinkel) [inline src/open-scd.ts:138]: Could we use a lambda here for stylistic consistency?

File: src/open-scd.ts [hunk 2]
```diff
@@ -130,11 +130,27 @@ export class OpenSCD extends Setting(
     this.saveUI.close();
   }
 
+  private formatXml(xml: string, tab?: string) {
```
Comment (c-dinkel) [inline src/open-scd.ts:133]: Do we use the configurable indentation anywhere?

File: src/open-scd.ts (hunk 1 above)
Comment (JakobVogelsang) [reply src/open-scd.ts:138]: For sure! Good idea

File: src/open-scd.ts (hunk 2 above)
Comment (JakobVogelsang) [reply src/open-scd.ts:133]: Not know at least. You are right. If we might need it we can add later.

**Review by JakobVogelsang — APPROVED** [review]
LGTM

**Pull request merged**